fake.go

```diff
@@ -519,6 +520,59 @@ func (fake *Fake) Dump() string {
 	return buf.String()
 }
 
+// ParseDump can parse a dump for a given nft instance.
+// It expects table name and the same nftables family in all rules based on a given fake.nftContext.
```

Comment:
```suggestion
// It expects fake's table name and family in all rules.
```

`nftContext` is an internal concept; shouldn't be in the docs